docs/architecture.md

```
@@ -320,52 +320,52 @@ public class Provider : AggregateRoot<Guid>
+        {
+            return Result<ProviderDto>.Failure(Error.NotFound("Provider.NotFound", "Fornecedor não encontrado"));
+        }
+
+        provider.UpdateProfile(command.Name, command.BusinessProfile.ToDomain(), command.UpdatedBy);
+        await uow.SaveChangesAsync(ct);
+
+        return Result<ProviderDto>.Success(provider.ToDto());
+    }
+}
+```
     }
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)